content/sensu-go/5.15/installation/install-sensu.md

```diff
@@ -346,12 +346,14 @@ Sensu keepalives are the heartbeat mechanism used to ensure that all registered
 
 To verify that the agent has registered with Sensu and is sending keepalive events, open the entity page in the [Sensu web UI][ui] or run `sensuctl entity list`.
 
-### Activate licensed-tier features
+### Activate commercial features
```

Comment:
I'm not sure the word "activate" make sense anymore, since users don't have to do anything to activate the commercial features. Perhaps, "access" makes more sense, since they must be using a supported distribution?

Reply:
I revised to just "Commercial features"